src/actions/index.js

```diff
@@ -125,8 +125,27 @@ export const fetchStorageData = () => {
   }
 }
 
+const tryUpdate = async (instance, { retries = 0 }) => {
+  let updatedInstance
+  try {
+    updatedInstance = await cozyFetch('PUT', '/settings/instance', instance)
```

Comment:
[nitpick] you can put `return await cozyFetch('PUT', '/settings/instance', instance)` here